package io.github.VyctorHugoCorreia.exception;

public class TimeNaoEncontradoException extends RuntimeException {

    public TimeNaoEncontradoException() {
        super("Time n√£o encontrado.");
    }

}
